### ISSUE TITLE
HTTP2 Support

### ISSUE THREAD

**Issue by 39ff** [koulab/cowitter#1]
```
HTTP/2 200 
cache-control: no-cache, no-store, must-revalidate, pre-check=0, post-check=0
content-disposition: attachment; filename=json.json
content-length: 2263
content-type: application/json;charset=utf-8
date: Tue, 12 Mar 2019 02:20:17 GMT
expires: Tue, 31 Mar 1981 05:00:00 GMT
last-modified: Tue, 12 Mar 2019 02:20:17 GMT
pragma: no-cache
server: tsa_m
status: 200 OK
strict-transport-security: max-age=631138519
x-access-level: read-write-directmessages
x-content-type-options: nosniff
x-frame-options: SAMEORIGIN
x-rate-limit-limit: 900
x-rate-limit-remaining: 889
x-rate-limit-reset: 1552357333
x-response-time: 133
x-twitter-response-tags: BouncerCompliant
x-xss-protection: 1; mode=block; report=https://twitter.com/i/xss_report
```

**Comment by 39ff** [comment]
https://tools.ietf.org/html/rfc7540#section-8.1.2.4
```
   For HTTP/2 responses, a single ":status" pseudo-header field is
   defined that carries the HTTP status code field (see [RFC7231],
   Section 6).  This pseudo-header field MUST be included in all
   responses; otherwise, the response is malformed (Section 8.1.2.6).

   HTTP/2 does not define a way to carry the version or reason phrase
   that is included in an HTTP/1.1 status line.
```

**Comment by 39ff** [comment]
patched https://github.com/39ff/cowitter/commit/96758adef97e121bfe7755be0dbb5a2346955be3
